### ISSUE TITLE
It's not compatible with API 23, please solve

### ISSUE THREAD

**Issue by azadso** [rey5137/material#302]
Recently, I've downloaded the library to use it in my app, I'm using API 23 and this is my .gradle file:

```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 23
    buildToolsVersion '23.0.0'

    defaultConfig {
        applicationId "com.standingtech.convertor"
        minSdkVersion 16
        targetSdkVersion 23
        versionCode 1
        versionName "1.0"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    compile fileTree(include: ['*.jar'], dir: 'libs')
    testCompile 'junit:junit:4.12'
    compile 'com.android.support:appcompat-v7:23.1.1'
    compile 'com.android.support:design:23.1.1'
    compile 'br.com.liveo:navigationdrawer-material:2.3.3'
    compile 'com.android.support:support-v4:23.1.1'
    compile 'com.android.support:cardview-v7:+'
}

**Comment by fernandodev** [comment]
@azadso it was fixed in PR #299 
I'm waiting for merge

**Comment by andrewpros** [comment]
Last commit was months ago, is there a better fork?

**Comment by jakewilson801** [comment]
This should be fixed. Im considering using this library over one I'm currently using but this is a show stopped please merge.

**Comment by rey5137** [comment]
I have released 1.2.2 version that fixed this bug.
